home/migrations/0083_alter_contentpage_whatsapp_body.py

```diff
@@ -0,0 +1,262 @@
+# Generated by Django 4.2.11 on 2024-11-04 14:59
```

Comment:
Have you tested this migration? I have a feeling that it will just delete all list items, since it doesn't know to default them to `next_message` list items. I think we need a manual migration step as part of this: https://docs.wagtail.org/en/v4.2/advanced_topics/streamfield_migrations.html#streamfield-data-migrations

Reply:
So the data isn't dropped, but it doesn't actually show up (e.g. if you undo the migration, everything goes back to normal).

I did try to do a manual migration step, but it never seemed to actually take. My thinking was that as a fallback we could do an export before upgrading the CMS version, then do an import after the upgrade.

Should I try the manual data migration step again? Maybe this time you can help solve it :-)